File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& v) {
        int sum=0,usum=0;
        
        for(int i=0;i<v.size();i++)
        {
            if(v[i]==1)
            {
                sum++;
                usum=max(sum,usum);
                
            }
            else
            {
                sum=0;
            }
        }
        return usum;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        max_count = 0

        for n in nums:
            if n == 1:
                count += 1
                max_count = max(max_count, count)
            else:
                count = 0
        return max_count